heroku.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:2.7-slim
+
+EXPOSE 8000
+WORKDIR /app
+CMD ["./bin/run-prod.sh"]
+
+RUN adduser --uid 431 --disabled-password --disabled-login --gecos 'webdev' --no-create-home webdev
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential libxml2-dev libxslt1-dev libffi-dev python-dev \
+    python-pip node-less locales libpq-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN dpkg-reconfigure locales && locale-gen C.UTF-8 && \
```

Comment:
Is this needed?

Reply:
It was already there.

Reply:
Agreed but I am not sure that it's needed :)